server/routes/api/case.js

```diff
@@ -22,12 +22,15 @@ router.post('/:caseId/stage/:stageId/allocate/team',
     allocateCaseToTeamMember,
     (req, res) => res.json({ redirect: '/' })
 );
-router.post(['/:caseId/stage/:stageId/entity/:entity/:context/:action', '/:caseId/stage/:stageId/entity/:entity/:action'],
-    getFormForCase,
-    fileMiddleware.any(),
-    processMiddleware,
-    validationMiddleware,
-    caseApiResponseMiddleware
+router.post([
+    '/:caseId/stage/:stageId/entity/:entity/:context/:action',
+    '/:caseId/stage/:stageId/entity/:entity/:context/:caseType/:action',
+    '/:caseId/stage/:stageId/entity/:entity/:action'],
+getFormForCase,
+fileMiddleware.any(),
+processMiddleware,
+validationMiddleware,
+caseApiResponseMiddleware
```

Comment:
Can you tidy up the formatting here as well please?

Reply:
The linter keeps complaining when I try anything else and 'corrects' it to this format.

Reply:
no worries... leave it as it is then